### DISCUSSION TITLE
Support for inertia js routing.

### DISCUSSION THREAD

**Discussion by msoubofficial** [quasarframework/quasar#15969] [Ideas / Proposals]
Hello , since quasar as vite plugin is love , but what most of the laravel inertia js vue progs require is support for the inertia Link component provided with inertia js vue adapter. No doubt quasar components work fine but the navigation is still a simple issue that stops the user base from using quasar in their app.

**Comment by SergBudkin** [comment]
Any ideas ?

**Comment by stefanvanherwijnen** [comment]
I don't use Inertia and your question is kind of vague, but I think you need: https://inertiajs.com/manual-visits

Instead of `to=` use `@click="router.visit(...)"`.
